cscs-checks/analytics/hpcg/hpcg_benchmark.py

```diff
@@ -0,0 +1,138 @@
+import reframe as rfm
+import reframe.utility.sanity as sn
+
+@rfm.simple_test
+class HPCGCheckRef(rfm.RegressionTest):
+    def __init__(self):
+        super().__init__()
+
+        self.descr = 'HPCG reference benchmark'
+        self.valid_systems = ['daint:mc', 'daint:gpu', 'dom:gpu', 'dom:mc']
+        self.valid_prog_environs = ['PrgEnv-gnu']
+        self.modules = ['craype-hugepages8M']
+        self.build_system = 'Make'
+        self.build_system.options = ['arch=MPI_GCC_OMP']
+        self.sourcesdir = 'https://github.com/hpcg-benchmark/hpcg.git'
+
+        self.executable = 'bin/xhpcg'
+        self.executable_opts = ['--nx=104', '--ny=104', '--nz=104', '-t2']
+        output_file = sn.getitem(sn.glob('HPCG*.txt'), 0)
```

Comment:
Can you put a comment before this line explaining why this is necessary. It's not so straightforward for a first-time reader to understand why you are using filename globbing and you are picking only the first element. AFAIR, there is just a single file generated, but it's name is not known.